### PR TITLE
EDDSA Support

### PR DESCRIPTION
**Pull request by venafi-iw**
This PR is to support EDDSA signing/verification functionality.

### CODE REVIEW

File: const.go [hunk 1]
```diff
@@ -536,6 +536,7 @@ const (
 	CKM_ECMQV_DERIVE                     = 0x00001052
 	CKM_ECDH_AES_KEY_WRAP                = 0x00001053
 	CKM_RSA_AES_KEY_WRAP                 = 0x00001054
+	CKM_EDDSA                            = 0x00001057
```
Comment (miekg) [inline const.go:539]: is there a link for me to verify this number?

File: go.mod [hunk 2]
```diff
@@ -1,3 +1,3 @@
-module github.com/miekg/pkcs11
+module github.com/venafi-iw/pkcs11
```
Comment (miekg) [inline go.mod:1]: you're changing all import paths

File: const.go (hunk 1 above)
Comment (venafi-iw) [reply const.go:539]: https://docs.oasis-open.org/pkcs11/pkcs11-curr/v3.0/csprd01/pkcs11-curr-v3.0-csprd01.pdf

File: go.mod (hunk 2 above)
Comment (venafi-iw) [reply go.mod]: sorry, was doing testing, but this has been fixed in latest commit.

File: const.go (hunk 1 above)
Comment (miekg) [reply const.go:539]: why do we need to add this manually instead of updating pkcs11.h and regenning the go code?

Comment (venafi-iw) [reply const.go:539]: I'm open to either way, but if we go down the regen approach then there needs to be documented steps on how to do so, such as in a Makefile.

Comment (miekg) [reply const.go:539]: I'm generating it now directly with go generate, see #147 
Also opened #146 to upgrade wholesale to v3 of pkcs11 spec and then regen, seems easier than adding const piecemeal

**Pull request closed without merging**